apps/test/unit/gamelab/GameLabSpriteTest.js

```diff
@@ -0,0 +1,63 @@
+var P5 = require("script!@cdo/apps/../lib/p5play/p5");
+var P5Play = require("script!@cdo/apps/../lib/p5play/p5.play");
+import GameLabP5 from '@cdo/apps/gamelab/GameLabP5';
+import JSInterpreter from '@cdo/apps/JSInterpreter';
+import {injectJSInterpreter} from '@cdo/apps/gamelab/GameLabSprite';
+import sinon from 'sinon';
+import {expect} from '../../util/configuredChai';
+
+describe('GameLabSprite', function () {
+  var gameLabP5;
+
+  beforeEach(function () {
+    gameLabP5 = new GameLabP5();
+    gameLabP5.init({onExecutionStarting: sinon.spy(), onPreload: sinon.spy(), onSetup: sinon.spy(), onDraw: sinon.spy()});
+    gameLabP5.startExecution();
+
+    var interpreter = {getCurrentState: function () {return {};}};
+    injectJSInterpreter(interpreter);
+  });
+
+  afterEach(function () {
+    gameLabP5.resetExecution();
+  });
+
+  describe('isTouching', function () {
+    it('returns false if the collider and colliding sprite dont overlap', function () {
+      var sprite1 = gameLabP5.p5.createSprite(0, 0, 100, 100);
+      var sprite2 = gameLabP5.p5.createSprite(200, 200, 100, 100);
+      var isTouching1to2 = sprite1.isTouching(sprite2);
+      var isTouching2to1 = sprite2.isTouching(sprite1);
+      expect(isTouching1to2).to.equal(false);
+      expect(isTouching1to2).to.equal(isTouching2to1);
```

Comment:
This can be simplified with some nice Chai BDD syntax.  [See docs here](http://chaijs.com/api/bdd/).

```javascript
expect(sprite1.isTouching(sprite2))
  .to.equal(sprite2.isTouching(sprite1))
  .and.to.be.false;
```